app/src/main/java/com/vinh/dictionary_1/data/model/DailyWord.java

```diff
@@ -0,0 +1,117 @@
+package com.vinh.dictionary_1.data.model;
+
+import android.arch.persistence.room.ColumnInfo;
+import android.arch.persistence.room.Entity;
+import android.arch.persistence.room.Ignore;
+import android.arch.persistence.room.PrimaryKey;
+import android.arch.persistence.room.TypeConverters;
+import android.support.annotation.NonNull;
+import com.vinh.dictionary_1.data.source.local.DictTypeConverter;
+
+/**
+ * Created by VinhTL on 06/11/2017.
+ */
+
+@Entity(tableName = "daily_word_tbl")
+public class DailyWord {
+    private String mType;
+    @NonNull
+    @PrimaryKey
+    @ColumnInfo(name = "word")
+    private String mWord;
+    @ColumnInfo(name = "description", typeAffinity = ColumnInfo.BLOB)
+    @TypeConverters(DictTypeConverter.class)
+    private String mDescription;
+    @ColumnInfo(name = "mean")
+    private String mShortDescription;
+    private String mPronounce;
+    @ColumnInfo(name = "dnpn", typeAffinity = ColumnInfo.BLOB)
+    @TypeConverters({ DictTypeConverter.class })
+    private String mAlikeWord;
+    @ColumnInfo(name = "word_ko_dau")
+    private String mWordWithoutDiacritic;
+    @Ignore
+    private boolean isBookmarked;
```

Comment:
WARNING: [AvoidFieldNameMatchingMethodName] Field isBookmarked has the same name as a method
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#AvoidFieldNameMatchingMethodName